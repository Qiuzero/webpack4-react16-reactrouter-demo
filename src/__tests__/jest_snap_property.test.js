it('检查匹配器并测试通过', () => {
  const user = {
    createAt: new Date(),
    id: Math.floor(Math.random() * 20),
    name: 'Durban',
  };

  expect(user).toMatchSnapshot({
    createAt: expect.any(Date),
    id: expect.any(Number),
  });
});
