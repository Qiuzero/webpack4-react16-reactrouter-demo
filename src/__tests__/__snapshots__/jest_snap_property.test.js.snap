// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`检查匹配器并测试通过 1`] = `
Object {
  "createAt": Any<Date>,
  "id": Any<Number>,
  "name": "Durban",
}
`;
